jina/serve/runtimes/gateway/http/models.py

```diff
@@ -228,14 +228,14 @@ class JinaRequestModel(BaseModel):
     Jina HTTP request model.
     """
 
+    # the dict one is only for compatibility.
+    # So we will accept data: {[Doc1.to_dict, Doc2...]} and data: {docs: [[Doc1.to_dict, Doc2...]}
     data: Optional[
         Union[
             PydanticDocumentArray,
             List[PydanticDocument],
```

Comment:
```suggestion
            List[PydanticDocument],
```

Do not use this type, it is unncessary